.ci/monolithic-windows.sh

```diff
@@ -45,4 +51,4 @@ cmake -S ${MONOREPO_ROOT}/llvm -B ${BUILD_DIR} \
       -D CMAKE_CXX_COMPILER_LAUNCHER=sccache
 
 echo "--- ninja"
-ninja -C ${BUILD_DIR} ${targets}
+ninja -C "${BUILD_DIR}" "${targets}"
```

Comment:
Could it be that adding quotes around `targets` breaks Windows build? For example [here](https://buildkite.com/llvm-project/github-pull-requests/builds/1606#018a8da6-0dda-4d24-a1cf-971e3450e4f8):

```
+ ninja -C C:/ws/src/build 'check-clang check-clang-tools check-flang check-lld check-llvm check-mlir check-polly'
ninja: error: unknown target 'check-clang check-clang-tools check-flang check-lld check-llvm check-mlir check-polly'
```

Reply:
on it

Reply:
fixed in #66192 , thanks for the headsup

Reply:
Thanks for the quick fix!